app/src/androidTestWebview/java/org/mozilla/focus/privacy/WebViewDataTest.java

```diff
@@ -218,14 +200,16 @@ public void DeleteWebViewDataTest() throws InterruptedException, UiObjectNotFoun
                 "/copper/truck/service-worker.js"); // Our service worker is installed
 
         // Now let's assert that there are no surprises in the data directory
-
         final File dataDir = new File(appContext.getApplicationInfo().dataDir);
         assertTrue("App data directory should exist", dataDir.exists());
 
         final File webViewDirectory = new File(dataDir, "app_webview");
         assertTrue("WebView directory should exist", webViewDirectory.exists());
-        assertEquals("WebView directory contains one subdirectory", 1, webViewDirectory.list().length);
-        assertEquals("WebView subdirectory is local storage directory", "Local Storage", webViewDirectory.list()[0]);
+        assertTrue("WebView directory contains several subdirectories", webViewDirectory.list().length <= 3);
+        assertTrue("WebView subdirectories may be one of several",
+                webViewDirectory.list()[0].equals("Service Worker")
+                        || webViewDirectory.list()[0].equals("GPUCache")
+                        || webViewDirectory.list()[0].equals("Local Storage"));
```

Comment:
i suppose i could do a webViewDirectory sort and then reliably just check for one of these, but it seemed better to explicitly call them all out at least as a record of all the subdirs that we expect to be here